<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1620</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1620</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1620</width>
    <height>1000</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="widgetHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1620</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1620</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1620</width>
     <height>100</height>
    </size>
   </property>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>0</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string>头像</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetFunc" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1620</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1620</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1620</width>
     <height>100</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ticketList">
      <property name="text">
       <string>车票查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addticketButton">
      <property name="text">
       <string>增加车次</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeticketbtn">
      <property name="text">
       <string>更改车次</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="userList">
      <property name="text">
       <string>用户列表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adduserButton">
      <property name="text">
       <string>添加用户</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="text">
       <string>保存修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetChart" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1620</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1620</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1620</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1620</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1620</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1620</width>
      <height>800</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page0">
     <widget class="QTableWidget" name="ticketWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1620</width>
        <height>700</height>
       </rect>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <column>
       <property name="text">
        <string>列车号</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>始发站</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>终点站</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发时间</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到站时间</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>车票数量</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>票价</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <widget class="QTableWidget" name="searchticketWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1620</width>
        <height>700</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>列车号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>始发站</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>终点站</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出发时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到站时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>车票数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>票价</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QTableWidget" name="userWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1620</width>
        <height>700</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>账号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>购票信息</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QTableWidget" name="searchuserwidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1620</width>
        <height>700</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>账号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>购票信息</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Choosewidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1620</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1620</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1620</width>
      <height>100</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QStackedWidget" name="stackedWidget1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>查询方式</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="addressButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>按车站</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="timeButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>按日期</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>出发地</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="baEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>目的地</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="eaEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>出发日期</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>到站日期</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDateTimeEdit" name="btEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QDateTimeEdit" name="etEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="searchticketbtn">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="searchuserbtn">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
